### ISSUE TITLE
LightGBM: A Highly Efficient Gradient Boosting Decision Tree

### ISSUE THREAD

**Issue by SEOJIN-Lab** [gdscewha-3rd/Study-PaperReview#8]
- 저자가 뭘 해내고 싶어했는가
    - 고차원, 큰 데이터 사이즈일때도 효율적이고 안정성있는 알고리즘 연구
    - LightGBM 실험
    
- 이 연구의 접근에서 중요한 요소는 무엇인가
    - Algorithm: GOSS, EFB
    - LightGBM experiment
    
      이론)정확도는 유지 또는 향상시키면서 속도를 향상시키는 알고리즘이다. 
        
        → 실험 결과 일치!

**Comment by binable43** [comment]
부스팅은 XGBoost만 익숙했는데 덕분에 새로운 알고리즘에 대해 알아갑니다. 잘 읽었습니다 :)

**Comment by ssongedev** [comment]
논문의 중요한 점을 간결하게 잘 정리하셔서 덕분에 이해가 잘됩니다! 한 수 배웁니다!! :)

**Comment by chaerin314** [comment]
잘 읽었습니다!:)

**Comment by minha62** [comment]
처음 들어보는 알고리즘인데 다음에 더 찾아보고 싶네요! 좋은 내용 감사합니다:)
